code/datums/preferences/preferences.dm

```diff
@@ -486,10 +488,16 @@ var/list/removed_jobs = list(
 
 				if (new_name)
 					src.name_last = new_name
-					src.real_name = src.name_first + " " + src.name_last
+					src.set_real_name()
 					src.profile_modified = TRUE
 					return TRUE
 
+			if ("update-hyphenation")
```

Comment:
Tiny nitpick: this should be called `toggle-hyphenation` as `update` implies it provides a value rather than toggling.